### ISSUE TITLE
Langchain client does not include Resource Group Header

### ISSUE THREAD

**Issue by pj-sap** [SAP/ai-sdk-js#183]
### Describe the Bug

When working with the langchain client, the set Resource Group is ignored in requests to the `chat/completions` endpoint. Retrieval of the deployment works properly though (as can be seen from the used endpoint).

Minimal code example:

```javascript
const chatClient = new AzureOpenAiChatClient({ modelName: 'gpt-4o', resourceGroup: 'my-resource-group' });
let resp = await chatClient.invoke('Hey, how are you?');
```

### Steps to Reproduce

1. Create a new resource group in AI Core (!= 'default')
2. Create a deployment for the resource group
3. Try to call the deployment through the langchain client
4. Inspect the request made to AI Core, the Resource Group Header is set to 'default'; therefore, the request returns with a 404 error

### Expected Behavior

The Resource Group is properly set in the `chat/completions` call.

### Used Versions

- Node version via `node -v`: 20.17.0
- NPM version via `npm -v: 10.8.2
- SAP Cloud SDK for AI version: 1.0.0

### Affected Development Phase

Getting Started

### Impact

Inconvenience

**Comment by tomfrenken** [comment]
Hi @pj-sap,

thanks for raising the issue, I've created a PR and we should be able to include the fix in our next release on Thursday.

**Comment by deekshas8** [comment]
Hi @pj-sap ,

The fix was released in the new minor version v1.1.0. Please try it out.

**Comment by pj-sap** [comment]
It works now. Thanks a lot!
